app/src/main/java/com/keylesspalace/tusky/util/StatusParsingHelper.kt

```diff
@@ -0,0 +1,62 @@
+/* Copyright 2022 Tusky Contributors
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>. */
+
+package com.keylesspalace.tusky.util
+
+import android.text.SpannableStringBuilder
+import android.text.Spanned
+import androidx.core.text.parseAsHtml
+
+/**
+ * parse a String containing html from the Mastodon api to Spanned
+ */
+fun String.toSpanned(): Spanned {
```

Comment:
I think the name `toSpanned()` on all Strings might be a bit confusing. I think it should donate that this is HTML and/or that we are parsing it.
`parseSpannedHtml()` or something like that? `parseMastodonHTML()`?

Reply:
true